Makefile

```diff
@@ -55,3 +64,30 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+.PHONY: bench-deps
+bench-deps:
+	@echo "Installing benchstat tool"
+	rm -rf ${BENCH_PATH}
+	mkdir ${BENCH_PATH}
+	go clean -testcache
+	cd ${BENCH_PATH} && git clone https://go.googlesource.com/perf && cd perf && go install ./cmd/benchstat
+	rm -rf ${BENCH_PATH}/perf
+
+.PHONY: bench
+${BENCH_FILE} bench:
+	@echo "Running benchmark tests from the current branch"
+	go test ${TEST_PATH} -bench=. -run=^Benchmark -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 \
+		| tee ${BENCH_FILE}
+	benchstat ${BENCH_FILE}
+
+${REFERENCE_FILE}:
+	@echo "Running benchmark tests from master for using results in bench-diff target"
+	cd ${BENCH_PATH} && rm -rf go-tarantool && git clone https://github.com/tarantool/go-tarantool.git
+	cd ${BENCH_PATH}/go-tarantool && go test ${BENCH_PATH}/go-tarantool/... -bench=. -run=^Benchmark -benchmem \
```

Comment:
I propose to put bench options to a variable too. the same options are used in two targets.

```diff
+++ b/Makefile
@@ -9,6 +9,7 @@ TEST_PATH ?= ${PROJECT_DIR}/...
 BENCH_FILE := ${BENCH_PATH}/bench.txt
 REFERENCE_FILE := ${BENCH_PATH}/reference.txt
 BENCH_FILES := ${REFERENCE_FILE} ${BENCH_FILE}
+BENCH_OPTIONS := -bench=. -run=^Benchmark -benchmem -benchtime=${DURATION} -count=${COUNT}
 
 .PHONY: clean
 clean:
@@ -77,17 +78,16 @@ bench-deps:
 .PHONY: bench
 ${BENCH_FILE} bench:
        @echo "Running benchmark tests from the current branch"
-       go test ${TEST_PATH} -bench=. -run=^Benchmark -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 \
-               | tee ${BENCH_FILE}
+       go test ${TEST_PATH} ${BENCH_OPTIONS} 2>&1 | tee ${BENCH_FILE}
        benchstat ${BENCH_FILE}
 
 ${REFERENCE_FILE}:
        @echo "Running benchmark tests from master for using results in bench-diff target"
        cd ${BENCH_PATH} && rm -rf go-tarantool && git clone https://github.com/tarantool/go-tarantool.git
-       cd ${BENCH_PATH}/go-tarantool && go test ${BENCH_PATH}/go-tarantool/... -bench=. -run=^Benchmark -benchmem \
-               -benchtime=${DURATION} -count=${COUNT} 2>&1 | tee ${REFERENCE_FILE}
+       cd ${BENCH_PATH}/go-tarantool && go test ${BENCH_PATH}/go-tarantool/... ${BENCH_OPTIONS} 2>&1 | tee ${REFERENCE_FILE}
        rm -rf ${BENCH_PATH}/go-tarantool
 
```

Reply:
Looks good, applied.